metabase/metabase
# Static viz path for the updated trend chart

Related to https://github.com/metabase/metabase/pull/37038

### Description

Creates a static visualization rendering path with a data contract similar to the application's data visualization contract to ensure maximum reuse of settings computation and actual viz rendering code.

In the scope of ECharts migration we did a similar change [here](https://github.com/metabase/metabase/pull/34820/commits/3f1ad8d6f1d9b5298a25f5533ab684ae835b7743#diff-98aa068e0239e92cdc383ef4a069b7563a7aed3e842509f83c24b9561b3260af) which we will rebase on top of the current PR change once it is merged. In the referred commit the rendering method is called `isomorphic` which was not the most precise name and `javascript_visualization` makes the code a bit more obvious.

### How to verify

1. Create a Trend chart
2. Send a test dashboard subscription with it
3. Ensure it renders the HTML placeholder

### Demo

<img width="636" alt="Screenshot 2024-01-08 at 10 49 12 PM" src="https://github.com/metabase/metabase/assets/14301985/ee8a5700-0f8c-4291-a5b2-bf337474af33">

### Checklist
